Header header
float32 timestamp
float32 roll_error
float32 pitch_error
int32 fix_type
int32 heading
int32 antena
int32 satelites
float32 hdop
float32 vdop
int32 odometer_active
float32 odometer_speed
float32 lat_stdev
float32 lon_stdev
float32 height_stdev
float32 heading_stdev
int32 data_overflow
float32 gnss_time_gap
